srcs/check_map.c

```diff
@@ -0,0 +1,106 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   check_map.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gudias <marvin@42lausanne.ch>              +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/08/18 17:40:48 by gudias            #+#    #+#             */
+/*   Updated: 2022/09/01 15:16:01 by gudias           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "cub3d.h"
+
+static int	save_player_info(t_info *info, int x, int y)
+{
+	if (info->player.x != -1.0f)
+		return (error_msg("More than 1 player"));
+	else
+	{
+		info->player.x = (float) x;
+		info->player.y = (float) y;
```

Comment:
The player is in the middle of the tile
```suggestion
		info->player.x = (float)x + 0.5f;
		info->player.y = (float)y + 0.5f;
```